### DISCUSSION TITLE
Fully offline backup

### DISCUSSION THREAD

**Discussion by CrisR82** [oobabooga/text-generation-webui#5710] [Q&A]
Probably a weird question, but I am quite curious - is there a way that once setup on a machine, it can be ZIP'ed and then even if the OS is reinstalled and you're fully offline, just extract it and use without ever connecting to the internet?

**Comment by TheLounger** [comment]
Oobabooga uses a conda environment that is completely stand-alone, as far as I can see. It doesn't even rely on the system-installed python and doesn't require you to install any packages.
So, as long as you have some recent GPU drivers, it should work fine.
I would only use an archive for backups though, you may be better off with an (encrypted) image that you can mount.

**Reply by CrisR82** [reply]
Well, that's kind'a the thing I'm going for - I very much value the existence of full offline availability of things I use, these days projects and repositories get taken down left and right for all softs of reasons so I want to be sure that even if this happens, I can just unzip a file and keep going. Guess this will do the trick for me then, thanks for the info ^-^
